agent/config.go

```diff
@@ -95,48 +90,52 @@ func (a Agent) identifyCurrentHost() error {
 		return agentError(err)
 	}
 
-
 	hostURL := index.Links.FindByRel("host-list")
 	hosts := []common.HostMessage{}
 	err = client.Get(hostURL, &hosts)
 	if err != nil {
 		return agentError(err)
 	}
+	log.Println("Retrieved hosts list, found", len(hosts), "hosts")
+
+	// find our host in the list of hosts
```

Comment:
This part isn't new behavior. It takes the list of hosts from topology then uses it to discover
1) what host this agent is responsible for, for iptables rules and routes to tap interfaces
2) what other hosts we need to reach, for inter-host routes.